apps/researcher/src/app/[locale]/object/[id]/page.tsx

```diff
@@ -0,0 +1,185 @@
+          ))}
+        </>
+      )}
+    </div>
+  );
+}
+
+interface Props {
+  params: {id: string};
+}
+
+export default async function Details({params}: Props) {
+  const id = decodeURIComponent(params.id);
+  const object = await objectFetcher.getById({id});
+  const t = await getTranslations('Details');
+
+  if (!object) {
+    return <div data-testid="no-object">{t('noObject')}</div>;
+  }
+
+  return (
+    <>
+      <Link href="/">{t('backButton')}</Link>
+      <div className="grid grid-cols-5 mt-5">
+        <div className="col-span-3 flex flex-col space-y-4">
+          <PageHeader>
+            <PageTitle id="about">{object.name}</PageTitle>
+          </PageHeader>
+          <div>
+            <H3>{t('description')}</H3>
+            <div>{object.description}</div>
+          </div>
+          <div>
+            <H3>{t('date')}</H3>
+            <NotImplemented />
+          </div>
+          <div>
+            <H3>{t('owner')}</H3>
+            {object.owner ? (
+              <a href={object.owner.id} target="_blank" rel="noreferrer">
+                {object.owner.name}
+              </a>
+            ) : (
+              <NoData />
+            )}
+          </div>
+          <div>
+            <H3>{t('handle')}</H3>
+            <NotImplemented />
+          </div>
+          <div>
+            <H3>{t('license')}</H3>
+            <NotImplemented />
+          </div>
+          <div className="space-y-4">
+            <H2>{t('moreInfo')}</H2>
+            <InfoBlock singleLine objectKey="materials" object={object} />
+            <InfoBlock singleLine objectKey="types" object={object} />
+            <InfoBlock
+              singleLine={false}
+              objectKey="techniques"
+              object={object}
+            />
+            <InfoBlock singleLine objectKey="creators" object={object} />
+            <InfoBlock
+              singleLine={false}
+              objectKey="inscriptions"
+              object={object}
+            />
+          </div>
+        </div>
+        <div className="col-span-2 flex flex-col space-y-4">
+          <div className="flex justify-end space-x-3">
+            <button
+              disabled
+              type="button"
+              className="rounded-md bg-white px-3 py-2 text-sm font-semibold text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 hover:bg-gray-50"
+            >
+              {t('saveButton')}
+            </button>
+            <button
+              disabled
+              type="button"
+              className="rounded-md bg-white px-3 py-2 text-sm font-semibold text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 hover:bg-gray-50"
+            >
+              {t('shareButton')}
+            </button>
+          </div>
+          {object.images?.length && (
+            <div className="relative col-span-2 h-[250px] w-full">
+              <Image
+                alt={object.name || object.id}
+                src={object.images[0].contentUrl}
```

Comment:
What should happen if an object has multiple images? The order of which the images are returned by the backend is undefined - we don't know whether the first one is the right/best one. @Doppen What do you think?

Reply:
Good question. We probably need an image gallery. [This](https://tailwindui.com/components/ecommerce/components/product-overviews#component-13a89c2dc50a31afd66541dc28fa3c13) for example.

I suggest adding a ticket for multiple images. So we can tackle this as a separate feature.

Reply:
https://github.com/colonial-heritage/colonial-collections/issues/new

Reply:
This question is also relevant for the object car (#81).